README.md

```diff
@@ -25,6 +25,10 @@ or
 yarn add @catchandrelease/arbor
 ```
 
+### To Start
+From the root directory of arbor run `yarn storybook`, then navigate to `localhost:6006` to see Arbor's documentation.
```

Comment:
The script should automatically open the browser and navigate to `localhost:6006`